### DISCUSSION TITLE
Modal from message service does not close when expected.

### DISCUSSION THREAD

**Discussion by Andrew-Seagle** [Megabit/Blazorise#3525] [Q&A]
I'm using the MessageAlert component and have even tried setting background cancel to true manually.
The modal is called on using MessageService.Warning and is not set up in any funky way from what I can tell.

No matter what I do besides press Ok or reload the app the modal doesn't hide.
That includes clicking outside of the modal, following links in the modal, or even navigating with the browser forward/back buttons.

Any help would be greatly appreciated!

**Comment by stsrki** [comment]
Do you have any errors in the console?

**Reply by Andrew-Seagle** [reply]
No errors in console or on the application side. 
Any ideas?

**Comment by stsrki** [comment]
It's hard to tell without any code. Any chance you could create a small project to reproduce it?

**Reply by Andrew-Seagle** [reply]
That would be pretty difficult for me to do at the moment, sorry.
I was wondering if maybe it had to do with the fade effect in the background?
I might just use a custom modal as currently only modals from the services are behaving this way.
Unfortunately, that may be faster than trying to diagnose this issue.

**Reply by stsrki** [reply]
It is possible that fade is affecting it. I will need to test it first to confirm if that is the case.
